string label
uint32 centerX
uint32 centerY
uint32 bWidth
uint32 bHeight
